packages/@coorpacademy-components/src/atom/autocomplete/index.js

```diff
@@ -16,42 +22,63 @@ const Autocomplete = props => {
     required,
     modified = false,
     error = false,
+    errorMessage,
     suggestions = [],
     onChange = noop,
     onFetch = noop,
     onClear = noop,
     onBlur = noop,
     onSuggestionSelected = noop,
-    title: propsTitle
+    title: propsTitle,
+    theme = 'default'
   } = props;
 
-  const title = `${propsTitle}${required ? ' *' : ''}`;
+  const mainClass = themeStyle[theme];
+  const title = `${propsTitle}${required ? '*' : ''}`;
   const className = getClassState(style.default, style.modified, style.error, modified, error);
```

Comment:
// *complete deps as suggested by eslint useMemo hooks
```suggestion
  const className = useMemo(() =>getClassState(style.default, style.modified, style.error, modified, error), []);
```